src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -135,19 +137,42 @@ public ElementTag getToggleSessionLevelInSectionButton() {
         
         return permissionInputGroup;
     }
-
+    
+    /**
+     * Creates a row of selections elements (checkboxes), each representing a single section
+     *   and automatically checks a single checkbox if special privileges have been assigned to
+     *   the section it corresponds to.
+     *   
+     * @param sectionNames the list of sections in the course
+     * @param sectionIndex the index of the section which will be checked for special privileges
+     * @return             a list of checkboxes, separated into rows and columns
+     */
     private List<List<ElementTag>> createSpecialSectionsForSectionRow(List<String> sectionNames, 
```

Comment:
Hmm they should, but maybe we should rename all of them as a separate issue then.